### PR TITLE
Create message thread

### PR DESCRIPTION
**Pull request by anbuihoang**
Create message thread

### CODE REVIEW

File: libs/components/src/lib/components/ChatWelcome/index.tsx [hunk 1]
```diff
@@ -7,21 +10,43 @@ export type ChatWelComeProp = {
...
 function ChatWelCome({ type, name, avatarDM }: ChatWelComeProp) {
+	const { currentThread } = useThreads();
+	const user = useSelector(selectMemberByUserId(currentThread?.creator_id as string));
 	return (
 		<div className="space-y-2 px-4 mb-4 mt-[250px]">
 			{type === 'CHANNEL' ? (
 				<div className="h-[75px] w-[75px] rounded-full bg-zinc-700 flex items-center justify-center pl-2">
 					<Hashtag defaultFill="#ffffff" defaultSize="w-10 h-10 mb-2" />
 				</div>
+			) : type === 'THREAD' ? (
```
Comment (phong9xNcc) [inline libs/components/src/lib/components/ChatWelcome/index.tsx]: please create enum

File: libs/components/src/lib/components/ChatWelcome/index.tsx [hunk 2]
```diff
@@ -7,21 +10,43 @@ export type ChatWelComeProp = {
...
 				<img className="h-[75px] w-[75px] rounded-full flex items-center justify-center" src={avatarDM} />
 			)}
-			<p className="text-xl md:text-3xl font-bold pt-1" style={{ wordBreak: 'break-word' }}>
-				{type === 'CHANNEL' ? 'Welcome to #' : ''} {name}
-			</p>
-			<p className="text-zinc-400 text-sm">
-				{type === 'CHANNEL' ? `This is the start of the #${name} channel.` : `This is the start of your conversation with ${name}`}
-			</p>
+
+			{type === 'THREAD' && currentThread && (
```
Comment (phong9xNcc) [inline libs/components/src/lib/components/ChatWelcome/index.tsx]: please create enum

Comment (anbuihoang) [reply libs/components/src/lib/components/ChatWelcome/index.tsx]: Done

File: libs/components/src/lib/components/ChatWelcome/index.tsx (hunk 1 above)
Comment (anbuihoang) [reply libs/components/src/lib/components/ChatWelcome/index.tsx]: Done

**Pull request merged**